GsaGHTests/1_BaseParameters/2_Geometry/GsaElement2dTest.cs

```diff
@@ -386,5 +400,52 @@ private GsaElement2d CreateSampleElement2dWithQuad4Type() {
 
       return elem;
     }
+
+    private GsaElement2d CreateLoadPanel() {
+      var pts = new Point3dList {
+       new Point3d(0, 0, 0),
+       new Point3d(2, 0, 0),
+       new Point3d(2, 2, 0),
+       new Point3d(0, 2, 0),
+     };
+
+      pts.Add(pts[0]);
+      var pol = new Polyline(pts);
+      var elem = new GsaElement2d(pol.ToPolylineCurve());
```

Comment:
This part is creating a GsaElement2D, we are doing this in multiple parts. Could we extract and re-use this? I noticed that sometimes you have different Point3ds, I don't understand which case are they testing?

if we need different points could we have the points as a specific list

```
var points = GetPlanarPoints();
var gsaElement2D = GetElement2dFromPoints(points)
```

Reply:
I have moved this in common place https://github.com/arup-group/GSA-Grasshopper/pull/695/files#diff-ff2742c0dcc2f3a0c413a11880749c0f04aca194050e993211639d8313fdf970